types/index.d.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare module "@symfony/stimulus-bridge" {
+
+    import { Application } from "stimulus";
```

Comment:
Shouldn't the import be at the root level? (not sure, I don't work much with TypeScript types)